src/components/scenes/TransactionListScene.tsx

```diff
@@ -248,10 +246,9 @@ function TransactionListComponent(props: Props) {
   })
 
   return (
-    <SceneWrapper hasNotifications hasTabs>
+    <SceneWrapper accentColor={iconColor} hasNotifications hasTabs>
       {({ insetStyles }) => (
         <>
-          <LinearGradient colors={[backgroundGradientColor, '#00000000']} start={{ x: 0, y: 0 }} end={{ x: 0, y: 1 }} style={StyleSheet.absoluteFill} />
```

Comment:
The design spec requires this remains:

https://app.asana.com/0/search?q=wallet%20colors&child=1205856947404193

<img width="1101" alt="Screenshot 2023-11-29 at 3 48 29 PM" src="https://github.com/EdgeApp/edge-react-gui/assets/517469/82b7995d-0faf-4451-bede-6498dcd117d7">